### ISSUE TITLE
Bug: Accuracy drop after convert

### ISSUE THREAD

**Issue by chenyuqi990215** [nerox8664/onnx2keras#119]
I use the following code to convert from Pytorch to Tensorflow however accuracy drop after convert.

```
    def pytorch_to_keras(
            self, model, input_shapes=None,
            change_ordering=True, verbose=False, name_policy=None,
    ):
        for name, parameters in model.named_parameters():
            print(name, ':', parameters.type())
        correct_cnt = 0
        tot_cnt = 0
        data_loader = build_dataset(DATASET_NAME,IS_TRAIN,128)
        for (image, label) in data_loader:
            pred = model(image).detach().cpu().numpy()
            pred = np.argmax(pred,axis=1)
            correct_cnt += np.sum(pred == label.numpy())
            tot_cnt += pred.shape[0]
        print(correct_cnt / tot_cnt)
        input = torch.randn([100, *image_size(DATASET_NAME)])
        # 设置输入张量名，多个输入就是多个名
        input_names = ["input"]
        # 设置输出张量名
        output_names = ["output"]
        # 自定义onnx文件名和路径
        onnx_filename = "model.onnx"
        # 执行转化和保存
        torch.onnx.export(model, input, onnx_filename, verbose=True, input_names=input_names,
                          output_names=output_names)

        onnx_model = onnx.load(onnx_filename)

        k_model = onnx_to_keras(onnx_model=onnx_model, input_names=["input"],
                                input_shapes=input_shapes, name_policy=name_policy,
                                verbose=verbose, change_ordering=change_ordering)
        pred = k_model.predict(self.x_test,batch_size=256)
        print(sum(np.argmax(pred,axis=1) == self.y_test) / len(self.y_test))
        return k_model
```

pkl_file in Pytorch version in given bellow:

[model.zip](https://github.com/nerox8664/onnx2keras/files/6330856/model.zip)

**Comment by MT010104** [comment]
Hi! I met the same problem. LeNet5's accuracy drops by almost 90% but VGG16‘s accuracy doesn't change. I've verified that it's not the data or the model itself that‘s wrong. Have you solved it?
